bitstyles/bitstyles/objects/_input.scss

```diff
@@ -86,10 +86,13 @@
 //
 // Styleguide 1.22.4
 
-.o-label + .o-input--checkbox {
+/* postcss-bem-linter: ignore */
+.#{$bitstyles-namespace}o-label + .#{$bitstyles-namespace}o-input--checkbox {
```

Comment:
Open for suggestions on a better way to solve this. We could add something like `o-input--checkbox-label` to any checkbox labels, and then change this line to `.#{$bitstyles-namespace}o-input--checkbox-label + .#{$bitstyles-namespace}o-input--checkbox {` but that feels ugly.